src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -247,6 +247,20 @@ When you're ready for the publication process, click "Publish" next to the integ
 
 ![Buttons providing the options to publish or delete.](publish-button.png)
 
+## Image Requirements
+
+Published integrations are required to have a logo image uploaded. The requirements for those are fairly rigid:
+* Logo size must be between 256x256 px and 1024x1024px
```

Comment:
I would change it to something like 
- `256px by 256px and 1024px by 1024px` or
- `256 &times; 256px and 1024 &times; 1024px` -> renders kinda prettier